astropy/time/core.py

```diff
@@ -464,6 +499,35 @@ def __repr__(self):
     def __str__(self):
         return str(getattr(self, self.format))
 
+    def strftime(self, format_spec):
+        """
+        Convert Time to a string according to a format specification.
+        See time.stftime documentation for format specification.
```

Comment:
Should be `time.strftime`.  You should make this a link, which I think can be done with a Sphinx intersphinx link like:
```
See `time.strftime` documentation ..
```

Reply:
I should link the python documentation here?

Reply:
we're using the python intersphinx, so these will automatically turned into links. `time` is part of the standard library, so I think this will work as Tom suggested above (if you get build errors, let me know and I can have a more detailed look at it)